internal/epoch/epoch_manager.go

```diff
@@ -1014,15 +1014,23 @@ func rangeCheckpointBlobPrefix(epoch1, epoch2 int) blob.ID {
 }
 
 // NewManager creates new epoch manager.
-func NewManager(st blob.Storage, paramProvider ParametersProvider, compactor CompactionFunc, log logging.Logger, timeNow func() time.Time) *Manager {
+func NewManager(
+	st blob.Storage,
+	paramProvider ParametersProvider,
+	compactor CompactionFunc,
+	log logging.Logger,
+	timeNow func() time.Time,
+	allowCleanupWritesOnIndexLoad bool,
```

Comment:
wiring `allowCleanupWritesOnIndexLoad` by passing it when an `epoch.Manager` is created.